docs/platforms/javascript/common/session-replay/troubleshooting.mdx

```diff
@@ -17,6 +17,10 @@ This guide aims to extend the <PlatformLink to="/troubleshooting/">main troubles
 
 Canvas is supported in SDK versions >= `7.98.0`. Please see the <PlatformLink to="/session-replay/#canvas-recording">canvas setup documention</PlatformLink> to get started with canvas recordings.
 
+## My `canvas` has been tainted
+
+Set `crossorigin="anonymous"` on your images. This will allow images that are loaded from foreign origins to be used in `canvas` as if they have been loaded from the current origin. See the [CORS documention](https://developer.mozilla.org/en-US/docs/Web/HTML/CORS_enabled_image) for more information about this error.
```

Comment:
Maybe link directly to the `Security and tainted canvases` header https://developer.mozilla.org/en-US/docs/Web/HTML/CORS_enabled_image#security_and_tainted_canvases